tests/autobahn/Dockerfile.aiohttp

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.1
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN \
+    apt-get update && \
+    apt-get -y install --no-install-recommends wait-for-it && \
```

Comment:
Why is this necessary? It looks like this is never used in this specific container.